electron/logger.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  createLogger,
+  format
+} from 'winston'
+import DailyRotateFile from 'winston-daily-rotate-file'
+// Need real-world data on how big these logs realistically get so the maxsize of them aren't just guesses (right they are)
+const heroicCrashes = new DailyRotateFile({
+  datePattern: 'YYYY-MM-DD',
+  dirname: './logs/heroic',
+  filename: 'crashes-%DATE%.log',
+  maxFiles: '2d',
+  maxSize: '2m'
+})
+const heroicDebug = new DailyRotateFile({
+  datePattern: 'YYYY-MM-DD',
+  dirname: './logs/heroic',
+  filename: 'debug-%DATE%.log',
+  level: 'debug',
+  maxFiles: '1d',
+  maxSize: '10m'
+})
+const heroicErrors = new DailyRotateFile({
+  datePattern: 'YYYY-MM-DD',
+  dirname: './logs/heroic',
+  filename: 'errors-%DATE%.log',
+  level: 'warn',
+  maxFiles: '5d',
+  maxSize: '3m'
+})
+const heroicInfo = new DailyRotateFile({
+  datePattern: 'YYYY-MM-DD',
+  dirname: './logs/heroic',
+  filename: 'info-%DATE%.log',
+  level: 'info',
+  maxFiles: '2d',
+  maxSize: '5m'
+})
+
+const Logger = createLogger({
+  exceptionHandlers: [
+    heroicCrashes
+  ],
+  exitOnError: false,
+  format: format.combine(
+    format.timestamp({
+      format: 'YYYY-MM-DDTHH:mm:ss:SSZZ'
```

Comment:
```suggestion
      format: 'HH:mm:ss:SSZZ'
```

Ugh I'm getting confused myself now.
Here's an example what I want: `23:45:06.420-0100`

Reply:
we good now